aws/resource_aws_lb_ssl_negotiation_policy.go

```diff
@@ -139,16 +139,14 @@ func resourceAwsLBSSLNegotiationPolicyRead(d *schema.ResourceData, meta interfac
 		return fmt.Errorf("Unable to find policy %#v", getResp.PolicyDescriptions)
 	}
 
-	// We can get away with this because there's only one policy returned
-	policyDesc := getResp.PolicyDescriptions[0]
-	attributes := flattenPolicyAttributes(policyDesc.PolicyAttributeDescriptions)
-	d.Set("attributes", attributes)
-
 	d.Set("name", policyName)
 	d.Set("load_balancer", lbName)
 	d.Set("lb_port", lbPort)
 
-	return nil
+	// We can get away with this because there's only one policy returned
+	policyDesc := getResp.PolicyDescriptions[0]
+	attributes := flattenPolicyAttributes(policyDesc.PolicyAttributeDescriptions)
+	return d.Set("attribute", attributes)
```

Comment:
This attribute is going to be very complicated, if not impractical to fix unfortunately. The ELB service automatically expands the results to include all SSL attributes (unordered, so we'd need to switch to `TypeSet` anyways), which we would be quite impractical to force practitioners to write out and potentially update each time the API updates. 😦 For example from running the acceptance testing:

```
--- FAIL: TestAccAWSLBSSLNegotiationPolicy_basic (95.19s)
testing.go:684: Step 0 error: After applying this step and refreshing, the plan was not empty:
DIFF:
DESTROY/CREATE: aws_lb_ssl_negotiation_policy.test
attribute.#:        "93" => "7" (forces new resource)
attribute.0.name:   "ADH-AES128-GCM-SHA256" => "AES128-GCM-SHA256" (forces new resource)
attribute.0.value:  "false" => "true" (forces new resource)
attribute.1.name:   "ADH-AES128-SHA" => "ECDHE-RSA-AES128-GCM-SHA256" (forces new resource)
attribute.1.value:  "false" => "true" (forces new resource)
attribute.10.name:  "ADH-RC4-MD5" => "" (forces new resource)
attribute.10.value: "false" => "" (forces new resource)
```

My suggestion for now would be just remove or comment out the `d.Set()` with a note mentioning this API expansion issue.